integration/tests/bft/gossipsub/signature/requirement/orchestrator.go

```diff
@@ -107,11 +106,7 @@ func (s *Orchestrator) sendUnauthorizedMsgs(t *testing.T) {
 // This func allows us to ensure that unauthorized messages have been processed.
 func (s *Orchestrator) sendAuthorizedMsgs(t *testing.T) {
 	for i := 0; i < numOfAuthorizedEvents; i++ {
-		event := bft.RequestChunkDataPackEgressFixture(s.T, s.attackerVNWithMsgSigning, s.victimENID, insecure.Protocol_PUBLISH)
-		err := s.OrchestratorNetwork.SendEgress(event)
-		require.NoError(t, err)
-		s.authorizedEvents[event.FlowProtocolEventID] = event
-		s.authorizedEventReceivedWg.Add(1)
+
```

Comment:
The BFT integration test where this is used is failing (probably because of this omission):

https://github.com/onflow/flow-go/actions/runs/6300677258/job/17109591746?pr=4720

```
--- FAIL: TestGossipSubSignatureRequirement (24.58s)
    --- FAIL: TestGossipSubSignatureRequirement/TestGossipSubSignatureRequirement (5.00s)
FAIL
FAIL	github.com/onflow/flow-go/integration/tests/bft/gossipsub/signature/requirement	24.635s
```

Reply:
https://github.com/onflow/flow-go/pull/4720/commits/8d8c54ed128ec7550ebeb6d056337a726ea8e819 unintended